runtime/common/src/lib.rs

```diff
@@ -212,10 +212,11 @@ macro_rules! create_runtime {
 
                 // Money
                 Balances: pallet_balances::{Call, Config<T>, Event<T>, Pallet, Storage} = 10,
-                TransactionPayment: pallet_transaction_payment::{Config, Pallet, Storage} = 11,
-                Treasury: pallet_treasury::{Call, Config, Event<T>, Pallet, Storage} = 12,
-                Vesting: pallet_vesting::{Call, Config<T>, Event<T>, Pallet, Storage} = 13,
-                MultiSig: pallet_multisig::{Call, Event<T>, Pallet, Storage} = 14,
+                Bounties: pallet_bounties::{Call, Event<T>, Pallet, Storage} =  11,
+                TransactionPayment: pallet_transaction_payment::{Config, Pallet, Storage} = 12,
+                Treasury: pallet_treasury::{Call, Config, Event<T>, Pallet, Storage} = 13,
+                Vesting: pallet_vesting::{Call, Config<T>, Event<T>, Pallet, Storage} = 14,
+                MultiSig: pallet_multisig::{Call, Event<T>, Pallet, Storage} = 15,
```

Comment:
It is better if we keep the indices in tact, that way the providers of signing software and hardware have less hassle to update their software / devices to mirror the new indices.

```suggestion
                TransactionPayment: pallet_transaction_payment::{Config, Pallet, Storage} = 11,
                Treasury: pallet_treasury::{Call, Config, Event<T>, Pallet, Storage} = 12,
                Vesting: pallet_vesting::{Call, Config<T>, Event<T>, Pallet, Storage} = 13,
                MultiSig: pallet_multisig::{Call, Event<T>, Pallet, Storage} = 14,
                Bounties: pallet_bounties::{Call, Event<T>, Pallet, Storage} =  15,
```

Reply:
To complement my comment, it's not just a hassle to update, this also leads to a breaking change for all signing device. For example, what the signing device assumed to be `Vesting`, suddenly became `Treasury`.